app/src/main/res/values-ja/strings.xml

```diff
@@ -199,8 +217,21 @@
     
     <string name="no_content">下書きはありません</string>
     <string name="action_save_one_toot">下書きに保存しました！</string>
-    
+
+    <!--These are for timestamps on statuses. For example: "16s" or "2d"-->
+    <string name="abbreviated_in_years">%d年後</string>
+    <string name="abbreviated_in_days">%d日後</string>
+    <string name="abbreviated_in_hours">%d時間後</string>
+    <string name="abbreviated_in_minutes">%d分後</string>
+    <string name="abbreviated_in_seconds">%d秒後</string>
+    <string name="abbreviated_years_ago">%d年前</string>
+    <string name="abbreviated_days_ago">%d日前</string>
+    <string name="abbreviated_hours_ago">%d時間前</string>
+    <string name="abbreviated_minutes_ago">%d分前</string>
+    <string name="abbreviated_seconds_ago">%d秒前</string>
+
     <string name="follows_you">あなたをフォロー中</string>
-    <string name="pref_title_alway_show_sensitive_media">常にすべての閲覧注意なコンテンツを表示する</string>
+    <string name="pref_title_alway_show_sensitive_media">閲覧注意のコンテンツを常に表示</string>
```

Comment:
I shortened this string because the original string is too long and automatically truncated in an actual device.